### PR TITLE
Updates to documentation for onboarding

### PR DESCRIPTION
**Pull request by kadrlica**
Updates to top-level Readme.md and GettingStarted.md to hopefully clarify the on-boarding process. Comments, suggestions, and direct edits are very welcome!

### CODE REVIEW

File: GettingStarted/GettingStarted.md [hunk 1]
```diff
@@ -1,40 +1,34 @@
...
-* [Notes on how to get set up on the LSST Science Platform (LSP) JupyterLab Notebook Aspect at the LSST Data Facility at NCSA](https://github.com/LSSTScienceCollaborations/StackClub/blob/master/GettingStarted/GettingStarted.md#accessing-the-lsst-science-platform)
+notebook aspect of the Rubin Science Platform (RSP). In these notes we provide:
+* [Notes on how to get set up on the Rubin Science Platform (RSP) JupyterLab Notebook Aspect at the LSST Data Facility at NCSA](https://github.com/LSSTScienceCollaborations/StackClub/blob/master/GettingStarted/GettingStarted.md#accessing-the-lsst-science-platform)
 * [Help with getting set up to run and edit the Stack Club tutorial notebooks](https://github.com/LSSTScienceCollaborations/StackClub/blob/master/GettingStarted/GettingStarted.md#running-and-contributing-to-the-stack-club-notebooks)
 
+## Accessing the Rubin Science Platform
+The [Rubin Science Platform (RSP) Notebook Aspect Documentation](https://nb.lsst.io/) provides an introduction to the system, including how to gain access and then how to use JupyterLab once you are in. 
+Accessing the RSP involves getting an NCSA account and figuring out VPN access.
```
Comment (MelissaGraham) [inline GettingStarted/GettingStarted.md:12]: I think we need to add a statement like "RSP access is for individuals with data rights as described in ls.st/rdo-013" here.

File: GettingStarted/GettingStarted.md [hunk 2]
```diff
@@ -1,40 +1,34 @@
...
-The [LSST Science Platform (LSP) Notebook Aspect Documentation](https://nb.lsst.io/) provides an introduction to the system, including how to gain access and then how to use JupyterLab once you are in.
-Getting on to the LSP involves getting an NCSA account, and then figuring out VPN access.
-
-#### Getting an LSST Science Platform Account
-The Stack Club has a limited number of active LSST Science Platform accounts it can support. To join the Stack Club and request one of these accounts, please fill out the [Stack Club Membership Application Form](https://goo.gl/forms/588KlPTFfkEEFFUu2). You'll need to agree to abide by the [Rules](../Rules.md), and then provide your full name (first and last) and your email address. If your application is successful, you'll get an email with instructions on how to set up your LSP account. 
+#### Getting a Rubin Science Platform Account
+To join the Stack Club and request one of these accounts, please fill out the [Stack Club Membership Application Form](https://forms.gle/rehWtaoHgiBx6VfZ6). You'll need to agree to abide by the [Rules](../Rules.md), provide your full name (first and last), and your email address.
```
Comment (MelissaGraham) [inline GettingStarted/GettingStarted.md:15]: We should add a data rights statement to the Rules as well.

**Review by MelissaGraham — APPROVED** [review]
Yes, all these changes look good to me. We just need to add a statement about how individuals must have Rubin data rights (RDO-013) in order to join Stack Club and have an RSP account. There are a few places where such a statement should go, I embedded some comments.

**Comment by kadrlica** [comment]
Thanks @MelissaGraham. Should we also add the data rights notice to the form itself? Maybe make applicants tick a checkbox?
https://forms.gle/rehWtaoHgiBx6VfZ6

**Comment by kadrlica** [comment]
Thanks @MelissaGraham. I've added the data rights statement in the following places:
* On the main README.md where we discuss joining the Stack Club
* On the GettingStarted.md where we talk about accessing the RSP
* At the start of Rules.md
* As a required question on the application form

Did I cover all the bases?

**Comment by MelissaGraham** [comment]
> Thanks @MelissaGraham. I've added the data rights statement in the following places:
> 
> * On the main README.md where we discuss joining the Stack Club
> * On the GettingStarted.md where we talk about accessing the RSP
> * At the start of Rules.md
> * As a required question on the application form
> 
> Did I cover all the bases?

Yes that's all good!

**Pull request merged**